### PR TITLE
53638: Add entry to intro to note pool widgets in dashboard change colors

### PR DESCRIPTION
**Pull request by Mrt134**
HTML build test: no issues.

Specific change to Angular GUI; no port to master branch needed.

### CODE REVIEW

File: userguide/intro.rst [hunk 1]
```diff
@@ -410,6 +410,10 @@ U2
   :menuselection:`Reporting`
   now shows the compressed physical L2ARC size.
 
+* The pool widgets in the
```
Comment (wblock) [inline userguide/intro.rst]:
```suggestion
* Pool widgets in the
```

**Comment by dlavigne** [comment]
No mention at all in the Storage chapter? Inquiring minds will wonder which color represents what (which is specified in the acceptance criteria for this ticket).

**Comment by wblock** [comment]
We can't mention specific colors, because colors are themed and none of them should be hard-coded due to allowing for color-blind users.  Someone using a black-and-white high contrast theme will not see the same colors as with other themes, and every theme can have different emphasis or "problem" colors.

**Comment by dszidi** [comment]
A couple of things to note...
- The Storage pool widgets specifically have colors changing based on status (red, yellow & green) so I think it would be worth mentioning
- It is entirely possible to put in some overrides to make the high contrast theme have some hard coded values. To accommodate color blind users, those alternate colors can be mentioned if need be

**Comment by Mrt134** [comment]
I was hesitant to put in the specific colors because they are pretty different between each theme, but I'll add some text to the storage chapter to try and explain it.

**Comment by Mrt134** [comment]
https://github.com/freenas/freenas-docs/pull/656

**Pull request merged**